setup/bindir/cloud-sysvmadm.in

```diff
@@ -72,21 +76,36 @@ do
         ;;
   z)    zone=" AND data_center_id=""$OPTARG"
         inzone=" in zone id=""$OPTARG"
+        ;;
+  i)    vmids="$OPTARG"
+        withids=" with id=""$OPTARG"
+        ;;
+  j)    vmtable="$OPTARG"
   esac
 done
 
 
-
+prapare_ids_clause() {
+  if [[ ! -z "$vmidsclause" ]]; then
+    return
```

Comment:
it looks line 89 to line 91 are not required.

ideally to have option to upgrade VR with livepatch.

Reply:
@weizhouapache since same method is called twice, added this check to not generate vmidsclause twice

Reply:
ok, makes sense @shwstppr